CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+- Updated dependencies and code to be in line with Gleam 1.0.
+- Reformat with Gleam 1.1.
+- Enable testing against the Javascript target in the CI.
```

Comment:
I think the changelog should mostly be for user facing things. "Updated dependencies" is really the only bit out of all of this with any externally observable impact.

Reply:
I hope it is more clear now.